On Darwin <= 9 - https://trac.macports.org/ticket/64913
builder error: Header file <sys/errno.h>, needed for module `sync` was not found.
Please install the corresponding development headers.

Header only required for specific concurrency example
https://github.com/vlang/v/commit/8f6fec4364b5eaf00a5ae07c70517cc3fc74e33b

--- vlib/sync/sync_darwin.c.v.original	2022-05-15 19:39:42.000000000 +0100
+++ vlib/sync/sync_darwin.c.v	2022-05-15 19:39:39.000000000 +0100
@@ -7,7 +7,6 @@
 
 #flag -lpthread
 #include <semaphore.h>
-#include <sys/errno.h>
 
 [trusted]
 fn C.pthread_mutex_init(voidptr, voidptr) int
